python/ppml/test/bigdl/ppml/resources/people.csv

```diff
@@ -0,0 +1,101 @@
+name,age,job
```

Comment:
You had better generate these files to a templete folder when you run your test, and delete them when the test finished.